docs/specs/disable-user.md

```diff
@@ -0,0 +1,37 @@
+# Disable User
+
+  * [Disabling user](#disabling-user)
+  * [Disabled user session](#disabled-user-session)
+  * [Disabled status](#disabled-status)
+  * [Future works](#future-works)
+
+## Disabling user
+
+A user can be disabled by admins. A disabled user cannot login, and appropriate
+error message would be shown when login is attempted.
+
+Admin may optionally provide a reason when disabling user. This reason would be
+shown when user attempted to login.
+
+## Disabled user session
+
+When a user is disabled, the user can no longer create new sessions. However,
+there may be existing logged in sessions.
+
+The authentication tokens (i.e. sessions, access token, refresh tokens, etc.)
```

Comment:
Do we update all token at once or do we check user disabled status every time we resolve a token?

Reply:
I intended to check disabled status for every resolver request for accuracy. Consumer may cache the resolver result/use planned JWT access token for performance.